common/services/user.js

```diff
@@ -76,7 +75,32 @@ function getNomisLocations(token) {
     .then(agencies => getLocationsByNomisAgencyId(agencies))
 }
 
+function getSupplierId(token) {
```

Comment:
I am not happy about this function - it's basically a copy of getAuthLocations, except it returns the supplier ID. Therefore introduces another API call to getSupplierByKey.

Any thoughts on the best way to proceed?